src/components/Image/_image.scss

```diff
@@ -0,0 +1,30 @@
+// ----------------------------------
+// UI Components
+// Image
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+.cc-media {
```

Comment:
is the plan to update all the selectors to use this prefix?

Reply:
Still mulling that one over. It may not be necessary for majority of components in the ac.uk ecosystem but it does mean we have much more scope for including them elsewhere in future. Perhaps we can put this one to group vote

Reply:
It's a yes from me 😁

Reply:
nuts to it, let's just do it moving forward

small PRs please 😆